app/containers/Dashboard/txnsToList.js

```diff
@@ -115,38 +111,18 @@ function infoIcon(event) {
   );
 }
 
-function txDescription(event, tableAddrs, address) {
+function txDescription(event, tableAddrs) {
   if (tableAddrs.indexOf(event.address) > -1) {
     return (
       <FormattedMessage
         key="descr"
         {...(event.type === 'income' ? messages.tableLeave : messages.tableJoin)}
       />
     );
-  } else if (isPowerUpEvent(event)) {
-    return 'Power Up';
-  } else if (isABPPayoutEvent(event)) {
-    return (
-      <FormattedMessage
-        key="descr"
-        {...messages.powerDownPayoutStatus}
-      />
-    );
-  } else if (event.address === confParams.pwrAddr && event.type === 'outcome') {
-    return (
-      <FormattedMessage
-        key="descr"
-        {...messages.powerUpStatus}
-      />
-    );
   } else if (isETHPayoutEvent(event)) {
     return <FormattedMessage key="descr" {...messages.ethPayoutStatus} />;
   } else if (isSellEvent(event)) {
     return <FormattedMessage key="descr" {...messages.sellStatus} />;
-  } else if (isPurchaseEndEvent(event, address)) {
-    return <FormattedMessage key="descr" {...messages.purchaseEnd} />;
-  } else if (isPurchaseStartEvent(event)) {
-    return <FormattedMessage key="descr" {...messages.purchaseStart} />;
   }
 
   return <FormattedMessage key="descr" {...messages.transferStatus} />;
```

Comment:
@sunify Can `ethPayoutStatus`, `sellStatus`, and `transferStatus` be removed from txnsToList.js?

Reply:
Yep